docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[data-quality-dash]]
+= Data Quality dashboard
+
+The Data Quality dashboard shows you whether your data is effectively mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
```

Comment:
```suggestion
The Data Quality dashboard shows you whether your data is correctly mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
```

Reply:
Both likely work but I feel like "correctly" is the slightly more accurate term to use. If I'm understanding this dashboard's purpose, it's to help users determine whether their data is ECS-_compliant_, meaning their data needs to fit into a defined set of rules. WDYT @andrew-goldstein @michaelolo24? Does my suggested change create too much of a closed statement?